trigger_circleci_pipeline.sh

```diff
@@ -8,21 +8,24 @@ CIRCLE_WORKFLOW_URL_BASE="https://circleci.com/workflow-run"
 
 branch="${BRANCH_REF#refs/heads/}"
 workflow_to_trigger=$1
-triggered_key=${2:-""}
 
 trigger_pipeline_response=$(
+    if [ ! $(jq 'has("triggered_workflow")' <<< "${WORKFLOW_KWARGS}") = false ]; then
+        echo "Workflow kwargs should not have an arg named 'triggered_workflow'"
+	exit 1
+    fi
+    request_data=$(
+    	jq \
+	--argjson triggered_workflow \""${workflow_to_trigger}"\" \
```

Comment:
I think you can use `--arg` instead so you won't need to escape it to a valid JSON string